core/types/src/attributes.rs

```diff
@@ -36,15 +36,15 @@ impl Attributes {
     /// # Arguments
     ///
     /// * `features_mask` - The mask to be set to the `xfrm` in the attributes
-    pub fn set_extended_features_mask(mut self, features_mask: u64) -> Self {
-        self.0.xfrm = features_mask;
+    pub fn set_extended_features_mask(mut self, features_mask: ExtendedFeatureRequestMask) -> Self {
+        self.0.xfrm = features_mask.bits();
         self
     }
 }
 
 impl Display for Attributes {
     fn fmt(&self, f: &mut Formatter<'_>) -> core::fmt::Result {
-        match Flags::from_bits(self.0.flags) {
+        match AttributeFlags::from_bits(self.0.flags) {
```

Comment:
`Flags`, with no qualifier coming out of `mc-sgx-core-types` felt ambiguous so added `Attribute` prefix